### ISSUE TITLE
[Bug]: The Franta Can Case - cans falling through the map - Antistasi towing

### ISSUE THREAD

**Issue by Bob-Murphy** [official-antistasi-community/A3-Antistasi#2543]
The Antistasi towing system creates invisible Franta cans to work `Land_CAN_V2_F`.
It happens on a semi-regular basis that the towing system seems to create / leave behind invisible franta cans which are falling through the map. These leads to spamming the RPT.
Adding it to the garbageClean is not an option as that could potentially mess with the towing, hence a different cleanup solution is required.

**Comment by killerswin2** [comment]
I'm very tempted to just remove towing in antistasi, as the script is kinda performance heavy.

**Comment by HakonRydland** [comment]
towing is critical to vehicle acquisition for single player, so i wouldnt do that, investigating better ways might be good though

**Comment by killerswin2** [comment]
Yeah there probably is a better system then just gutting.

**Comment by jaj22** [comment]
I had a brief look at this. Looks like the can only exists when the far end of the rope is either attached to a vehicle or carried by a player. It does however persist when the attached object is deleted, so that's probably the case that occasionally spams the RPT.

Loose ropes might not be great either though. They seem to be continually simulated (and will actually drift across the floor) and so should ideally be cleaned up.

**Comment by killerswin2** [comment]
Yeah, I noticed that. I was going to write a handler for deleted vehicles. But I'm more in favor of writing something else to remove the execVM attached to the code.

**Comment by MeltedPixel** [comment]
Linking #2650 as a new system should be used to fix this issue.
